great-idea/css/index.css

```diff
@@ -63,4 +63,173 @@ h1, h2, h3, h4, h5 {
     margin-bottom: 15px;
 }
 
-/* Copy and paste your work from yesterday here and start to refactor into flexbox */
\ No newline at end of file
+/* Copy and paste your work from yesterday here and start to refactor into flexbox */
+
+.container {
+	width: 900px;
+	margin: 0 auto;
+
+}
+title {
+	display: none;
+}
+h4{
+	margin-top: 25px;
+}
+h3 {
+	margin-top: 25px;
+	font-size: 27px;
+}
+
+header {
+	display: flex;
+	justify-content: space-between;
+	width: 100%;
+	padding-bottom: 50px;
+	padding-top: 25px;
+	align-items: center;
+}
+header nav {
+	flex-wrap: nowrap;
+	display: flex;
+	width: 75%;
+	justify-content: space-between;
+}
+header nav a{
+	color: gray;
+	text-decoration: none;
+	align-self: center;
+	margin-right: 50px;
+}
+header nav a:hover {
+	background-color: royalblue;
+	color: white;
+}
+header .logo {
+	width: 20%;
+}
+.lower_header {
+	padding-bottom: 40px;
+	border-bottom: 1px solid black;
+	line-height: 1.3;
+}
+.services_header_img {
+	margin-top: -25px;
+	padding-bottom: 10px;
+	width: 100%;
+}
+
+
+.slogan {
+	display: flex;
+	border-bottom: 2px solid black;
+	font-family: Bangers, cursive;
+	font-size: 55px;
+	text-align: center;
+	padding-bottom: 80px;
+}
+.slogan .left {
+	width: 60%;
+	align-self: center;
+}
+.slogan button {
+	font-size: 12px;
+	padding: 6px 28px;
+	background: transparent;
+	border: 1px black solid;
+}
+.slogan button:hover {
+	background-color: royalblue;
+	color: white;
+}
+
+
+.content {
+	display: flex;
+	flex-flow: row wrap;
+	padding-bottom: 40px;
+}
+.block1 {
+	width: 425px;
+	margin-right: 20px;
+	padding-bottom: 25px;
+	line-height: 1.2;
+}
+.block2 {
+	width: 275px;
+	margin-right: 18px;
+	line-height: 1.2;
+}
+.block_left {
+	width: 425px;
+	margin-right: 25px;
```

Comment:
Just a quick suggestion.  On your services page, you could have put the 6 cards in your content section in the same class, removed the margin right or left and added a `justify-content: space-between` in your `.content` class.  This way you would dry up your code and not repeat the same css.